components/style/matching.rs

```diff
@@ -453,6 +453,15 @@ trait PrivateMatchMethods: TElement {
                                        &new_values,
                                        None);
 
+            // If computed values for custom properties changed, we should cascade these changes to
+            // children (custom properties are all inherited).
+            // https://www.w3.org/TR/css-variables/#defining-variables
+            // (With Properties & Values, not all custom properties will be inherited!)
+            if old_values.as_ref().map(|v| v.as_ref()).and_then(|v| v.custom_properties()) !=
+               new_values.custom_properties() {
+                child_cascade_requirement = ChildCascadeRequirement::MustCascadeDescendants
```

Comment:
Initially was going to put it there, but it seemed to me like this wasn't really a style difference (?), i.e., couldn't find which ServoRestyleDamage flag corresponded to it, because any visible changes will show up there as changes to standard properties anyhow. WDYT?